mappings/net/minecraft/util/BiMapPalette.mapping

```diff
@@ -0,0 +1,25 @@
+CLASS brk net/minecraft/util/BiMapPalette
```

Comment:
Consider putting the palette classes in util/palette and profiler classes in util/profiler. Don't let util get too big.

Reply:
Done